sample-compose/app/src/main/AndroidManifest.xml

```diff
@@ -16,7 +16,7 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
 
-        <activity android:name=".ui.screens.MainActivity">
+        <activity android:name=".ui.screens.MainActivity" android:exported="true">
```

Comment:
@doannimble Would you mind remove these permission:
 ```
    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
    <uses-permission android:name="android.permission.CAMERA" />
```
I think we don't need them for now because if we still keep them, we will get the dialog for access permission when first install 🙏 What do you think @Tuubz

Reply:
@Wadeewee This was added for sample purposes: #261 🙏